start_area:
   x1: 1.75
   y1: -2.5
   x2: 2.25
   y2: -2
goal_area:
   x1: -2.25
   y1: 1
   x2: -1.25
   y2: 2
